File: ドキュメント/05_1対15比較図/rewrite.py
import re

file_path = '/Users/takumi.ikeda/運動タイプ診断/ドキュメント/05_1対15比較図/GFMC_盛り上げ隊長_vs_その他15タイプ.md'
with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

# We will just replace the sections.

File: ドキュメント/05_1対15比較図/rewrite.js
import fs from 'fs';

const filePath = '/Users/takumi.ikeda/運動タイプ診断/ドキュメント/05_1対15比較図/GFMC_盛り上げ隊長_vs_その他15タイプ.md';
let content = fs.readFileSync(filePath, 'utf-8');

// I will use an LLM call or write a detailed prompt for myself to generate these.
// Since I am an AI, I can just generate the content and replace it.
